components/02-elements/dropdown-list/dropdown-list.js

```diff
@@ -1,85 +1,87 @@
-        dropdownContent = dropdownItem.querySelector(`.${contentClass}[data-content="${dropdownId}"]`),
-        dropdownBlock   = item.closest('.dropdown-list');
+  function resetMqMediumOpen(item) {
+    const dropdownItem = item.parentNode,
+          dropdownContent = dropdownItem.querySelector(`.${contentClass}`);
 
-  if (!isMediumOpen(dropdownBlock)) {
-    if (dropdownContent.clientHeight > 0) {
-      dropdownContent.style.height = 0;
+    if (window.matchMedia(mq).matches) {
+      dropdownContent.style.height = 'auto';
       dropdownItem.classList.remove(openClass);
       setAriaAttributes(item, dropdownContent, true);
     }
     else {
-      dropdownContent.style.height = `${setListHeight(dropdownContent)}px`;
-      dropdownItem.classList.add(openClass);
+      dropdownContent.style.height = 0;
+      dropdownItem.classList.remove(openClass);
       setAriaAttributes(item, dropdownContent, false);
     }
   }
-  else {
-    dropdownContent.style.height = 'auto';
-    dropdownItem.classList.remove(openClass);
-    setAriaAttributes(item, dropdownContent, false);
-  }
-}
 
-dropdownItem.forEach(
-  key => key.addEventListener('click', (e) => {
-    e.preventDefault();
-    toggleContent(key, false);
-  }, false)
-);
+  function toggleContent(item) {
+    const dropdownId      = item.dataset.dropdown,
+          dropdownItem    = item.parentNode,
+          dropdownContent = dropdownItem.querySelector(`.${contentClass}[data-content="${dropdownId}"]`),
+          dropdownBlock   = item.closest('.dropdown-list');
 
-window.addEventListener('resize', () => {
-  const dropdownMediumOpen = document.querySelector(`.${mqClass}`);
-  if (dropdownMediumOpen) {
-    const dropdownItems =  [ ...dropdownMediumOpen.querySelectorAll(dropdownCollapseLabel)];
-    dropdownItems.forEach(key => resetMqMediumOpen(key));
+    if (!isMediumOpen(dropdownBlock)) {
+      if (dropdownContent.clientHeight > 0) {
+        dropdownContent.style.height = 0;
+        dropdownItem.classList.remove(openClass);
+        setAriaAttributes(item, dropdownContent, true);
+      }
+      else {
+        dropdownContent.style.height = `${setListHeight(dropdownContent)}px`;
+        dropdownItem.classList.add(openClass);
+        setAriaAttributes(item, dropdownContent, false);
+      }
+    }
+    else {
+      dropdownContent.style.height = 'auto';
+      dropdownItem.classList.remove(openClass);
+      setAriaAttributes(item, dropdownContent, false);
+    }
   }
-});
+
+  dropdownItem.forEach(
+    key => key.addEventListener('click', (e) => {
+      e.preventDefault();
+      toggleContent(key, false);
+    }, false)
+  );
+
+  window.addEventListener('resize', () => {
+    const dropdownMediumOpen = document.querySelector(`.${mqClass}`);
+    if (dropdownMediumOpen) {
+      const dropdownItems =  [ ...dropdownMediumOpen.querySelectorAll(dropdownCollapseLabel)];
+      dropdownItems.forEach(key => resetMqMediumOpen(key));
+    }
+  });
+})();
```

Comment:
did you refactor `dropdown-list-nested.js` too? or it's not needed?

Reply:
no, I only added Self-Invoking Anonymous Function, nothing more. There was an error so I had to add that.